packages/html/stories/Wires.stories.js

```
@@ -606,9 +606,9 @@ const Template = ({ label, ...args }) => {
 
   CellRenderer.registerShape('resistor', ResistorShape);
 
-  // Imlements a custom resistor shape. Direction currently ignored here.
+  // Implements a custom resistor shape. Direction currently ignored here.
 
-  EdgeStyle.WireConnector = function (state, source, target, hints, result) {
+  WireConnector = function (state, source, target, hints, result) {
```

ESLint

[error] 'WireConnector' is not defined.

(no-undef)